concepts/55 Vue Components/00 DevExtreme Vue Components.md

```diff
@@ -10,10 +10,10 @@ Try out DevExtreme components in your projects with our [30-day trial](https://j
     
     - [Add DevExtreme](/concepts/55%20Vue%20Components/05%20Add%20DevExtreme%20to%20a%20Vue%20Application/00%20Add%20DevExtreme%20to%20a%20Vue%20Application.md '//Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/') to your Vue application.
     
-    - Create [a new application with a template](/concepts/55%20Vue%20Components/02%20Create%20a%20DevExtreme%20Application.md '/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/') for quick setup.
+    - Create [a new application with a template](/Documentation/Guide/Vue_Components/Application_Template/) for quick setup.
 
 2. **Learn Basics**    
-See the [component configuration syntax](/concepts/55%20Vue%20Components/20%20Component%20Configuration%20Syntax '/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/').
+Learn the [first steps](/Documentation/Guide/Common/First_Steps/) and see the [component configuration syntax](/concepts/55%20Vue%20Components/20%20Component%20Configuration%20Syntax '/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/').
 
```

Comment:
Can you update the following lines below?

Line 19:
`Examine [over 450 technical demos](https://js.devexpress.com/Demos/WidgetsGallery) you can edit and from which you can copy code.`

Line 33:
`The latest DevExtreme release aligns with the latest Vue version. Check which specific Vue versions [are compatible with](/concepts/55%20Vue%20Components/80%20Supported%20Versions.md '/Documentation/Guide/Vue_Components/Supported_Versions/') your DevExtreme components.`